client/app/components/faq/faqdetails.js

```diff
@@ -1,41 +1,27 @@
-import React, { Component } from 'react';
+import React from 'react';
 import {
-	AppRegistry,
 	Text,
 	View,
 } from 'react-native';
-import BottomNavigation, { Tab, NavigationComponent } from 'react-native-material-bottom-navigation'
-import Icon from 'react-native-vector-icons/Entypo'
-import { TabNavigator, StackNavigator, HeaderBackButton } from 'react-navigation'
```

Comment:
All this was not being used lol.

Reply:
I think this was intended for use later by the previous team but never got implemented.  We will need to implement the back button too, so let's take note of these.